[TLT20IMPL-8] Personal account

OlegPozhenskij opened the pull request:
Ready for review

OlegPozhenskij committed:
To pull request

OlegPozhenskij commented on server/src/main/java/com/pozhenskii/springjwt/controllers/TestController.java:
```diff
@@ -0,0 +1,57 @@
+
+	@PutMapping("/updatePhoto")
+	public void updatePhoto(User user) {
+		userRepository.save(user);
```
Забыл сделать возврат request'а со различными статусами, сделаю

alexfoxrussian commented on client/src/App.vue:
```diff
@@ -1,39 +1,84 @@
-    <div class="app__content">
-      <router-view/>
+  <div id="app">
+    <nav class="navbar navbar-expand navbar-dark bg-dark">
```
Navigation Bar в отдельный компонент вынести надо

alexfoxrussian commented on client/src/services/user.service.js:
```diff
@@ -0,0 +1,25 @@
+import authHeader from './auth-header';
+
+// const API_URL = 'https://secure-class-app.herokuapp.com/api/test/';
+const API_URL = 'http://localhost:8080/api/test/';
```
API_URL - локалхост надо поменять

dnsonin replied:
/api/test/ - здесь 
в остальных URL так же: используй относительный путь.

alexfoxrussian commented on server/src/main/java/com/pozhenskii/springjwt/SpringBootSecurityJwtApplication.java:
```diff
@@ -0,0 +1,13 @@
+package com.pozhenskii.springjwt;
```
переименуй pozhenskii в classregister везде

dnsonin replied:
package com.netcracker.ncedu.tlt.impl.class_register;

OlegPozhenskij committed:
NavBar as a separate component,
API_URL as a relative path,
package as com.netcracker.ncedu.tlt.impl.class_register;

OlegPozhenskij committed:
A lot of changes in front-end.

alexfoxrussian commented on client/src/router.js:
```diff
@@ -1,32 +1,133 @@
+      component: Billboard
+    },
+    {
+      path: '/findStudent',
```
в URL не используют camelCase. либо все маленькие буквы, либо kebab-case.

OlegPozhenskij replied:
Исправил

alexfoxrussian commented on client/src/router.js:
```diff
@@ -1,32 +1,133 @@
+      component: Chat
+    },
+    {
+      path: '/passWord',
```
аналогично (либо маленькие буквы, либо kebab-case) хотя в данном случае - маленькие буквы.

OlegPozhenskij replied:
Исправил

alexfoxrussian commented on client/src/router.js:
```diff
@@ -1,32 +1,133 @@
+      path: '/billboard',
+      name: 'billboard',
+      // lazy-loaded
+      component: () => import('./views/Billboard.vue')
```
Определись, либо lazy-loading, либо нет. Зачем дублируешь руты?

OlegPozhenskij replied:
Видимо хотел убрать позже, исправил

alexfoxrussian commented on client/src/router.js:
```diff
@@ -1,32 +1,133 @@
+      path: '/findStudent',
+      name: 'findStudent',
+      // lazy-loaded
+      component: () => import('./views/FindStudent.vue')
```
аналогично верхнему

OlegPozhenskij replied:
Исправил

alexfoxrussian commented on client/src/router.js:
```diff
@@ -1,32 +1,133 @@
+      path: '/passWord',
+      name: 'passWord',
+      // lazy-loaded
+      component: () => import('./views/PassWord.vue')
```
аналогично верхнему замечанию.
+ password - это одно слово и camelCase'ом не пишется. (поправь везде)

OlegPozhenskij replied:
Исправил

alexfoxrussian reviewed (changes requested):
![image](https://user-images.githubusercontent.com/6068138/106262471-81ca2300-623c-11eb-9145-48209527cbbe.png)
зачем в МР добавил удаление презентаций?

OlegPozhenskij committed:
Add some fio, button, and validation in front-end.
Some changes in css.

OlegPozhenskij committed:
Some changes in router

dnsonin reviewed (approved):
Не вижу изменений в бэк-энде, OK.

OlegPozhenskij committed:
Add pdf

OlegPozhenskij committed:
Add pdf
